ynput/ayon-core
# Deadline: provide default user credential fields in studio settings

## Changelog Description
Some studios might have only single user/password credentials for communication with Deadline webservice. This PR provides new fields in `Studio Settings` which could be used as default values. These could be overridden by values provided by artists in `Site Settings` if necessary.

![image](https://github.com/ynput/ayon-core/assets/4457962/b316bff4-e6cd-4d72-8e68-f898ac24639a)

## Additional info
AYON secrets cannot be used as artist don't have access to those (and this runs in Tray). I am expecting pushback from customer though.

## Testing notes:
1. enable on Deadline > `Tools > Configure Repository Options > Web Service Settings > Require Authentication`
2. set passwor in Deadline > `Tools > Manage Users > Webservice password`
3. build and install Deadline addon (remove existing unzipped from `AppData`)
4. provide credentials in `ayon+settings://deadline/deadline_urls` (could be same as from https://github.com/ynput/ayon-core/pull/200 - just delete them first from `Site Settings` (on Studio page)
5. publish something to DL, check output from `Collect Deadline User Credentials`
